lib/ivis_opengl/pieblitfunc.cpp

```diff
@@ -256,7 +256,7 @@ static void pie_DrawRect(float x0, float y0, float x1, float y1, PIELIGHT colour
 		std::swap(y0, y1);
 	}
 	const auto& center = Vector2f(x0, y0);
-	const auto& mvp = defaultProjectionMatrix() * glm::translate(Vector3f(center, 0.f)) * glm::scale(x1 - x0, y1 - y0, 1.f);
+	const auto& mvp = defaultProjectionMatrix() * glm::translate(Vector3f(center, 0.f)) * glm::scale(glm::vec3{ x1 - x0, y1 - y0, 1.f });
```

Comment:
A lot of your replacement lines use list initialization (`{ ... }`), but the existing code (and the rest of the project) does not. I'm not necessarily advocating for a change, but did want to point out the discrepancy with the existing coding style.